packages/migrate/src/commands/MigrateDev.ts

```diff
@@ -325,8 +325,11 @@ ${chalk.green('Your database is now in sync with your schema.')}`,
           console.info() // empty line
           const successfulSeeding = await executeSeedCommand(seedCommandFromPkgJson)
           if (successfulSeeding) {
+            // TODO: what about Windows users running prisma on Git Bash or similar? Technically, they'd
+            // still be able to support emojis in their terminal.
```

Comment:
That is not a TODO in code, but a `kind/tech` issue to be created.

Reply:
Added https://github.com/prisma/prisma/issues/13355, I will remove the TODO in the code